lib/rspotify/base.rb

```diff
@@ -22,34 +23,36 @@ class Base
     #           tracks = RSpotify::Base.find(ids, 'track')
     #           tracks.class       #=> Array
     #           tracks.first.class #=> RSpotify::Track
-    def self.find(ids, type)
+    def self.find(ids, type, market: nil)
```

Comment:
Think it would be more useful if the param defaulted to the market specified in the configuration, and if that is not provided, then default to nil.